lib/chai.js

```diff
@@ -4,14 +4,13 @@
  * MIT Licensed
  */
 
-var used = []
-  , exports = module.exports = {};
+var used = [];
 
 /*!
  * Chai version
  */
 
-exports.version = '3.5.0';
+exports.version = require('../package').version;
```

Comment:
Just FYI our tooling automates writing this out as the string, so that we don't end up including all of the package.json in the browserify bundle.